tests/Avalonia.IntegrationTests.Appium/GestureTests.cs

```diff
@@ -147,9 +151,9 @@ public void RightTapped_Is_Raised_2()
             var device = new PointerInputDevice(PointerKind.Mouse);
             var b = new ActionBuilder();
 
-            b.AddAction(device.CreatePointerMove(border, 50, 50, TimeSpan.FromMilliseconds(100)));
+            b.AddAction(device.CreatePointerMove(border, 50, 50, TimeSpan.FromMilliseconds(50)));
             b.AddAction(device.CreatePointerDown(MouseButton.Right));
-            b.AddAction(device.CreatePointerMove(border, 2, 2, TimeSpan.FromMilliseconds(100)));
+            b.AddAction(device.CreatePointerMove(border, 52, 52, TimeSpan.FromMilliseconds(50)));
```

Comment:
I think this should have been: `CreatePointerMove(CoordinateOrigin.Pointer...` here, which should move the pointer relative to its current position.

Reply:
it says from the origin of the move... so I read that as origin of the border...

what I see on the screen is that with 52.. it moves 2 pixels


with 2 the test always fails.